site/js/expander.js

```diff
@@ -1,25 +1,14 @@
-// 	event handlers on each button
+// 	'See All' button event handlers on Documentation page
 //	on click hide elem, show all
 
-const buttons = document.querySelectorAll('button[id^="expandButton-"]');
-let i;
-
-for (i = 0; i < buttons.length; i++) {
-  buttons[i].addEventListener('click', function () {
-
-    const classId = this.id.replace('expandButton-', '');
-    const expanded = document.getElementById('expanded-' + classId);
-    const collapsed = document.getElementById('collapsed-' + classId);
-
-    if (expanded.className === 'none') {
-      expanded.className = 'inline';
-      collapsed.className = 'none';
-      this.textContent = 'See fewer';
-    } else if (expanded.className === 'inline') {
-      expanded.className = 'none';
-      collapsed.className = 'inline';
-      this.textContent = 'See all';
+document.getElementById('Documentation').addEventListener('click', (e) => {
+  if (e.target && e.target.className.includes('expand-button')) {
```

Comment:
I like to separate JS selectors from CSS class names, so you could freely change the CSS without affecting the JS. I'd suggest applying a `data-*` attribute that the JS reads. E.g. if you use `data-expand-button` you could replace this line with `if (!e.target.getAttribute('data-expand-button') return`;